cmd/gobl/sign.go

```diff
@@ -116,3 +106,22 @@ func (opts *signOpts) runE(cmd *cobra.Command, args []string) error {
 
 	return enc.Encode(env)
 }
+
+func loadPrivateKey(file string) (*dsig.PrivateKey, error) {
```

Comment:
I extracted this into a method because I needed to do the same from the serve command code. However, I don't know if it is a good practice or not to keep this function in this file considering and having the code in `serve.go` to access to it. Should I move it to some other place?

Reply:
It's fine here as it's related to signatures. Private methods are generally acceptable anywhere. I guess if this file started getting large, it could be extracted into a new file.